### PR TITLE
COMMERCE-11085

### PR DESCRIPTION
**Pull request by Brunno0liveira**
[COMMERCE-11085](https://issues.liferay.com/browse/COMMERCE-11085)

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductProductRelations.testcase [hunk 1]
```diff
@@ -55,6 +55,53 @@ definition {
...
+			AssertElementPresent(locator1 = "Button#TRASH_ENABLED");
+		}
+
+		task ("Then the product is successfully deleted") {
+			ClickNoError(locator1 = "Button#TRASH_ENABLED");
+
+			CommerceNavigator.searchEntry(entryName = "U-Joint");
+
+			AssertElementNotPresent(
+				key_entryName = "ABS Sensor",
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductProductRelations.testcase:100]: You deleted U-Joint but made an assertion for ABS Sensor, how did the test pass on your local?

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductProductRelations.testcase [hunk 2]
```diff
@@ -55,6 +55,53 @@ definition {
...
+
+			CommerceRelations.addProductRelation(productNameRelation = "U-Joint");
+
+			AssertElementPresent(
+				key_productName = "U-Joint",
+				key_value = "Up-Sell",
+				locator1 = "CommerceEntry#VALUE_PRODUCT_RELATIONS");
+		}
+
+		task ("When the admin delete the related product") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductProductRelations.testcase:82]: deletes

**Review by baharturk — CHANGES_REQUESTED** [review]
Please take a look. Thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:forward

**Comment by Brunno0liveira** [comment]
ci:stop:commerce-functional

**Comment by Brunno0liveira** [comment]
ci:forward

**Pull request closed without merging**